model/api_operations.js

```diff
@@ -75,11 +127,8 @@ function create_channel(user_id)
 }
 
 function get_next_video(channel_id)
-// TODO: fix function after correct model is added
 {
-    return Promise.resolve([channel_id, null]);
-    /*
-    return model.?????.findOne
+    return model.video.findOne
```

Comment:
Note also that there is probably some bug on GitHub, I see the last commit is pull request #7 on this branch which has `vote` column deleted(https://github.com/dxwc/Internet-Living-Room/blob/a3607dfa45e77e067a7cc79bd390478bf9c8f27f/model/setup.js#L119) as we discussed (if `vote` column is kept in `video` model and not somewhere separate, we wouldn't know if a particular user has already voted). Somehow the change has reappeared on this branch, but the diff tool here isn't showing it as addition (https://github.com/dxwc/Internet-Living-Room/blob/d5de936f866844a230221f98d81b8b5d5cfd74f8/model/setup.js#L119).

Reply:
i branch off from develop. But when i view the code on github, the `vote` is still there. Either way, I will remove it in my code.

Reply:
The vote data model that I did only keep records of what individual user vote for a particular video, it will be either -1 or 1. for -1 to be "downvote" and 1 to be "upvote". It doesn't have the total number of votes for videos. I guess we can do the math if we need the number, but it would easier if we have the vote column, then we can do increment or decrement on this column whenever users voted.